paperwork_backend/shell.py

```diff
@@ -3,20 +3,21 @@
 import json
 import os
 import sys
+import multiprocessing
```

Comment:
I forgot to ask you:
With `multiprocessing`, you're using entirely separate processes, which require serializing and serializing a lot of things. So why use `multiprocessing` instead of `threading` ?

Reply:
From the multiprocessing doc: `The multiprocessing package offers both local and remote concurrency, effectively side-stepping the Global Interpreter Lock by using subprocesses instead of threads. Due to this, the multiprocessing module allows the programmer to fully leverage multiple processors on a given machine.`
Regarding to [some](https://stackoverflow.com/a/2846695) [answers](https://stackoverflow.com/a/3046201) on stackoverflow I understand this `Global Interpreter Lock` to stop threads from `threading` to run on different CPUs in parallel.

Further coding for `multiprocessing` is in my opinion simpler than for `threading`.